ext.gradle

```diff
@@ -25,7 +25,7 @@
  * as we want to manage the versions in a single source.
  */
 
-final def SPINE_VERSION = '0.10.41-SNAPSHOT'
+final def SPINE_VERSION = '0.10.42.1-SNAPSHOT'
```

Comment:
Why not `43`?

Reply:
This is still WIP. In order to avoid conflicts with of my locally published versions with the globally accessible ones, I use non-typical versioning.